htdocs/core/modules/syslog/mod_syslog_file.php

```diff
@@ -173,7 +173,7 @@ public function export($content, $suffixinfilename = '')
 				$this->lastTime = $now;
 			}
 
-			$message = strftime("%Y-%m-%d %H:%M:%S", time()).$delay." ".sprintf("%-7s", $logLevels[$content['level']])." ".sprintf("%-15s", $content['ip'])." ".($this->ident > 0 ?str_pad('', $this->ident, ' ') : '').$content['message'];
+			$message = date("Y-m-d H:i:s", time()).$delay." ".sprintf("%-7s", $logLevels[$content['level']])." ".sprintf("%-15s", $content['ip'])." ".($this->ident > 0 ?str_pad('', $this->ident, ' ') : '').$content['message'];
```

Comment:
date() must be avoid because the problem of timezone is not clear. Use instead 
dol_print_date(dol_now(), "standard")